main/reference/zoe-api/zoe.md

```diff
@@ -197,9 +197,9 @@ these methods:
 const instance = await E(Zoe).getInstance(invitation);
 ```
 
-## E(Zoe).getProposalShapeForInvitation(invitation)
+## E(Zoe).getProposalShapeForInvitation(invitationHandle)
```

Comment:
yes, this matches [generated reference docs](https://agoric-sdk.pages.dev/interfaces/_agoric_zoe.src_zoeService_types_ambient.ZoeService#getProposalShapeForInvitation)

Reply:
p.s. here's hoping for the day when we don't have to maintain these docs in 2 places

 - https://github.com/Agoric/agoric-sdk/issues/4291